src/documentation/0016-npm/index.md

```diff
@@ -40,12 +40,15 @@ You can also install a specific package by running
 npm install <package-name>
 ```
 
+Furthermore, since npm 5, this command add `<package-name>` to the `package.json` file _dependencies_. Before version 5, you needed to add the flag `--save`.
```

Comment:
```suggestion
Furthermore, since npm 5, this command adds `<package-name>` to the `package.json` file _dependencies_. Before version 5, you needed to add the flag `--save`.
```